newIDE/app/src/Profile/SubscriptionDetails.js

```diff
@@ -13,43 +13,51 @@ type Props = {
 };
 
 export default ({ subscription, onChangeSubscription }: Props) =>
-  subscription && subscription.planId ? (
+  subscription ? (
```

Comment:
I think users that used to have a subscription but don't have it anymore may have a subscription object but with an empty planId.

Reply:
I think that's fine though? The logic on the `planId` is done below to display the subscription or not.
This check is mainly a proxy to a `loading` prop, to ensure the call is finished, so we can display the title

Reply:
You're right, that's totally fine to use this as a way to know if we're loading or not!